tutorials/mnist_tutorial_tf.py

```diff
@@ -21,6 +23,9 @@
 flags.DEFINE_integer('nb_epochs', 6, 'Number of epochs to train model')
 flags.DEFINE_integer('batch_size', 128, 'Size of training batches')
 flags.DEFINE_float('learning_rate', 0.1, 'Learning rate for training')
+flags.DEFINE_boolean('iterative', True, 'Whether to use basic iterative method'
```

Comment:
Maybe it make sense to have a string argument, something like "adversarial_method"?
Attacks would already have 3 methods: fgsm, saliency maps and basic iterative.
So having a sting flag with method type would be more generic approach.